### PR TITLE
Navigation now shows what page you are on

### PR DESCRIPTION
**Pull request by mojcas**
This is to address issue #44

### CODE REVIEW

**Comment by goranche** [comment]
impressive... :+1: 
will have a look  :wink:

File: TrackCat/templatetags/navigation.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from django import template
+from django.core import urlresolvers
+
```
Comment (goranche) [inline TrackCat/templatetags/navigation.py]: you have single spaces in these "empty" lines... might as well remove them :wink:

File: TrackCat/templatetags/navigation.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    except:
+        pass
+    matches = resolved and resolved.url_name == url_name
+    if matches and kwargs:
+        for key in kwargs:
+            kwarg = kwargs.get(key)
+            resolved_kwarg = resolved.kwargs.get(key)
+            if not resolved_kwarg or kwarg != resolved_kwarg:
+                return False
+    return matches
```
Comment (goranche) [inline TrackCat/templatetags/navigation.py]: missing newline at the end of the file... nothing major, but to be consistent... :smile:

**Comment by goranche** [comment]
small whitespace issues, nothing major, nice work :+1:

File: TrackCat/templatetags/navigation.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+from django import template
+from django.core import urlresolvers
+ 
+register = template.Library()
+
```
Comment (goranche) [inline TrackCat/templatetags/navigation.py]: in this line as well...

File: TrackCat/templatetags/navigation.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        pass
+    matches = resolved and resolved.url_name == url_name
+    if matches and kwargs:
+        for key in kwargs:
+            kwarg = kwargs.get(key)
+            resolved_kwarg = resolved.kwargs.get(key)
+            if not resolved_kwarg or kwarg != resolved_kwarg:
+                return False
+    return matches
+
```
Comment (goranche) [inline TrackCat/templatetags/navigation.py]: :smile: you've added a newline, but as there are spaces in the new line you created (don't mistake new line for a newline character) now again, there is no newline character at the end of the file...

File: TrackCat/templatetags/navigation.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+from django import template
+from django.core import urlresolvers
+ 
+register = template.Library()
+ 
+@register.simple_tag(takes_context=True)
+def current(context, url_name, return_value=' active', **kwargs):
+    matches = current_url_equals(context, url_name, **kwargs)
+    return return_value if matches else ''
+
```
Comment (goranche) [inline TrackCat/templatetags/navigation.py]: and this one... you don't need to delete the whole line, just the space characted at the start of it...
(the same was true about the other empty lines as well)

**Comment by goranche** [comment]
good work :+1:

**Comment by mojcas** [comment]
Thanks! :) :dart:

**Pull request merged**